library/src/text/quotes.rs

```diff
@@ -139,6 +139,8 @@ impl<'s> Quotes<'s> {
         let region = region.as_ref().map(Region::as_str);
         let (single_open, single_close, double_open, double_close) = match lang.as_str() {
             "de" if matches!(region, Some("CH" | "LI")) => ("‹", "›", "«", "»"),
+            "es" if matches!(region, Some("MX")) => return Self::default(),
+            "es" => ("“", "”", "«", "»"),
```

Comment:
As mentioned on discord, I think the optimal code here checks if the region is Some(“ES”) or None, then uses the fancy quotes, otherwise the code will fall back to the default.

Mexico isn’t the only other spanish-speaking country, after all, and it’s rather silly to include a full list of countries when we don’t need to.

Reply:
Done.